### PR TITLE
feat(ui): add onboarding empty state to storybook

### PR DESCRIPTION
**Pull request by ckj**
Adds an onboarding panel to storybook:

screen shot 2018-10-16 at 3 39 08 pm

### CODE REVIEW

File: docs-ui/components/emptyMessage.stories.js [hunk 1]
```diff
@@ -60,4 +60,25 @@ storiesOf('UI|EmptyMessage', module)
...
+  )
+  .add(
+    'in onboarding panel',
+    withInfo('Put this in a panel for maximum effect')(() => (
+      <Panel dottedBorder>
+        <EmptyMessage
+          size="large"
+          icon="icon-forward"
+          title="Your business intelligence workflow is missing crucial data"
+          description="Upgrade to the Large or Enterprise plan to send processed events to your favorite business intelligence tools such as Segment, Amazon&nbsp;SQS, and Splunk."
```
Comment (EvanPurkhiser) [inline docs-ui/components/emptyMessage.stories.js]: What's up with the non breaking space in here?

Comment (EvanPurkhiser) [reply docs-ui/components/emptyMessage.stories.js]: Oh since it's one word, nice

**Comment by ckj** [comment]
Note: i'm gonna followup after this PR with some general polish to emptyMessage

**Review by markstory — APPROVED** [review]
Nice work :clap:

**Comment by ckj** [comment]
Closing in favor of #10241

**Pull request closed without merging**